R/multi_clustering.R

```diff
@@ -116,7 +123,21 @@ multi_clust <- function(d, krange = 2:10, iter.max = 300, runs = 10,
   km[["clust_gap"]] <- cluster::clusGap(d, kmeans, 
                                         K.max = length(km[["clust_model"]]), 
                                         B = 15, verbose = FALSE)
-  km[["k_best"]] <- which.max(km[["sil_avg"]])
+  tryCatch({
+    nb_best <- suppressWarnings(suppressMessages(NbClust(d, min.nc = krange[1],
+                                        index = "alllong",
```

Comment:
I am going to change this index to 'all' once I merge this. As this is it terribly suggests that iris data set has 2 clusters by majority rule; changing to 'all' fixes that and reports 3 clusters. Plus it will be faster by default.